src/pages/Dashboard/Investment.tsx

```
@@ -0,0 +1,69 @@
+import { Table } from "../../components/Table"
+
+
+const Investment = () => {
+
+  const tableHead = ["S/N", "Amount Invested", "Expected Amount", "Completion Date","Status", ""]
+  const tableData:any = [
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)